tests/test_resource.py

```diff
@@ -0,0 +1,67 @@
+#  Copyright 2019 XLAB d.o.o.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import pytest
+from redfish_client.exceptions import BlacklistedValueException, TimedOutException
+
+
+def test_get_value_of_a_key(system_resource):
+    assert system_resource.ProcessorSummary.Status.State == 'Enabled'
+
+
+def test_get_invalid_key(system_resource):
+    with pytest.raises(KeyError):
+        value = system_resource.ProcessorSummary.Status.Invalid
```

Comment:
I suggest we group test_get_value_of_a_key and test_get_invalid_key under a class `TestGetKey` like this:

```python
class TestGetKey:
  def test_valid_key(system): ...
  def test_invalid_key(system): ...
```

This way you can easily add additional cases. Similarly, please group also other tests into classes.